packages/bridge/src/messaging/messaging.ts

```diff
@@ -0,0 +1,63 @@
+import { log } from "../logging";
+import {
+  BridgeError,
+  BridgeErrorCode,
+  BridgeRequest,
+  BridgeResponse,
+} from "../types";
+
+export function postMessage(message: BridgeRequest) {
```

Comment:
**question:** Is it planned to have two way communication via the bridge? 
As postMessage is a mechanism to post a message from an integration to the host app, the other way around is currently not handled, right?
When we add an integration to the Flutter app we might need a mechanism to tell the integration that it is now hosted within Flutter to e.g. hide certain parts of the UI. Depending on the setup queryParams, event listeners or a global window method come to my mind as possible mechanisms. Let's talk about this at 1 :)

Reply:
Sorry, should have documented that. The `makeRequest` function already listens for the response. In fact, every request should get a response from the host or flutter app. I did a possible implementation in the Console app bridge adapter: https://gitlab.com/flipcorp/flipnext/flip/-/blob/12576-create-basis-for-js-bridge/console/libs/shared/host/src/lib/bridge-adapter/bridge-adapter.service.ts